ext/random/random.c

```diff
@@ -633,7 +633,7 @@ PHPAPI uint64_t php_random_generate_fallback_seed(void)
 		/* Various PIDs. */
 		pid = getpid();
 		fallback_seed_add(&c, &pid, sizeof(pid));
-#ifndef WIN32
+#ifndef _WIN32
```

Comment:
The CSPRNG already uses PHP_WIN32: https://github.com/php/php-src/blob/bbbe56eecc6ff6b3971ba4f924d51c9aa11955c5/ext/random/csprng.c#L71

Can we unify this within ext/random?

Reply:
I think so, yes. Added. This getppid is not available on Windows. Let's go with PHP_WIN32 here then.